app/src/main/java/com/london/tudee/presentation/screens/category/category_details/CategoryDetailsDeleteBottomSheet.kt

```diff
@@ -20,38 +27,48 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.london.tudee.R
 import com.london.tudee.domain.entities.Category
-import com.london.tudee.presentation.components.bottom_sheet.TudeeBottomSheetScreen
 import com.london.tudee.presentation.components.buttons.TudeeNegativeButton
 import com.london.tudee.presentation.components.buttons.TudeeSecondaryButton
 import com.london.tudee.presentation.design_system.theme.ThemePreviews
 import com.london.tudee.presentation.design_system.theme.TudeeTheme
 import org.koin.compose.viewmodel.koinViewModel
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun CategoryDetailsDeleteScreen(
+fun CategoryDetailsDeleteBottomSheet(
     category: Category,
-    showBottomSheet: Boolean,
     modifier: Modifier = Modifier,
     onDismiss: () -> Unit,
     onCategoryDeleted: () -> Unit,
     onDeleteError: () -> Unit = {},
 ) {
-    TudeeBottomSheetScreen(
-        showBottomSheet = showBottomSheet,
-        modifier = modifier,
-        onDismiss = onDismiss,
-        screenContent = {},
-        bottomSheetActions = {},
-        bottomSheetContent = {
-            CategoryDetailsDeleteContent(
-                modifier = modifier,
-                category = category,
-                onCancel = onDismiss,
-                onCategoryDeleted = onCategoryDeleted,
-                onDeleteError = onDeleteError
-            )
-        }
-    )
+
+    ModalBottomSheet(
+        onDismissRequest = onDismiss,
+        sheetState = rememberModalBottomSheetState(
+            skipPartiallyExpanded = true
+        ),
+        windowInsets = WindowInsets(0),
+        dragHandle = {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(TudeeTheme.colors.surface),
+                contentAlignment = Alignment.Center
+            ) {
+                BottomSheetDefaults.DragHandle()
+            }
+        },
```

Comment:
You can pass the modifier directly to the `DragHandle()` instead of creating a new layout.